hashicorp/waypoint
# ui: upgrade typescript-eslint to 4.31.1

## Why the change?

When we upgraded typescript, I missed this notice from ESLint:

```
=============

WARNING: You are currently running a version of TypeScript which is not officially supported by @typescript-eslint/typescript-estree.

You may find that it works just fine, or you may not.

SUPPORTED TYPESCRIPT VERSIONS: >=3.3.1 <4.3.0

YOUR TYPESCRIPT VERSION: 4.4.2

Please only submit bug reports when using the officially supported version.

=============
```

The new version supports our version of Typescript.

## How do I test this?

No real need for manual testing of this one. If you want to be thorough, you can check out the branch, `yarn` and browse the code a bit in VS Code to see if any weird linting errors occur. Mostly though, this is a safe change that doesn’t impact the prod build at all.
